prowler/providers/gcp/gcp_provider.py

```diff
@@ -81,6 +106,54 @@ def __init__(
             client_id: str
             client_secret: str
             refresh_token: str
+
+        Raises:
+            GCPNoAccesibleProjectsError if no project IDs can be accessed via Google Credentials
+            GCPSetUpSessionError if an error occurs during the setup session
+            GCPLoadCredentialsFromDictError if an error occurs during the loading credentials from dict
+            GCPGetProjectError if an error occurs during the get project
+
+        Returns:
+            None
+
+        Usage:
+            - Authentication: Prowler will use by default your User Account credentials, you can configure it using:
+                - gcloud init to use a new account
+                - gcloud config set account <account> to use a specific account
+                - gcloud auth application-default login to use the Application Default Credentials
+                - Prowler will use the Application Default Credentials if no credentials are provided
```

Comment:
Since this belongs to the same method I'd combine them at the same level as `Using static credentials`

Reply:
Done

Reply:
Done now 😄